context/app/static/js/components/Detail/Summary/Summary.jsx

```diff
@@ -63,6 +64,7 @@ Summary.propTypes = {
   uuid: PropTypes.string.isRequired,
   description: PropTypes.string,
   status: PropTypes.string,
+  data_access_level: PropTypes.string,
```

Comment:
Is there a reason not to make this required?

Reply:
We're only passing the `data_access_level` prop in datasets.